compiler/tests/fixtures/features/aug_assign.fe

```diff
@@ -0,0 +1,57 @@
+contract Foo:
```

Comment:
It would be nice to have some tests that check that source files are correctly lowered. We need to be able to pretty-print ASTs for this, tho. See #326.